internal/rootcoord/util.go

```diff
@@ -135,3 +135,16 @@ func getTravelTs(req TimeTravelRequest) Timestamp {
 func isMaxTs(ts Timestamp) bool {
 	return ts == typeutil.MaxTimestamp
 }
+
+func getNeedChanNum(setNum int, chanMap map[typeutil.UniqueID][]string) int {
+	chanNames := typeutil.NewSet[string]()
+	for _, chanName := range chanMap {
+		chanNames.Insert(chanName...)
+	}
+
+	ret := chanNames.Len()
```

Comment:
what if chanMap is 
prefix-1, prefix-8, prefix-10?
I think we need to create at least 10 channels on that case